contracts/factory/PoolFactory.sol

```diff
@@ -0,0 +1,181 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/access/AccessControl.sol";
+import "@openzeppelin/contracts/governance/TimelockController.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+
+import "../openzeppelin/TransparentUpgradeableProxy.sol";
+
+import "./LibPoolConfig.sol";
+import "./LibFeeManager.sol";
+
+contract PoolFactory is Ownable, AccessControl {
```

Comment:
Is it common to use them together? My understanding is AccessControl can cover Ownable.

Reply:
You're right, will remove ownable and only use accessControl for permission.